core/src/main/scala/org/apache/spark/sql/streaming/SnappySinkCallback.scala

```diff
@@ -223,6 +223,11 @@ case class SnappyStoreSink(snappySession: SnappySession,
       ds.queryExecution.toRdd,
       StructType(ds.schema.fields))
   }
+
+  override def toString: String = {
+    val params = parameters + ("sinkCallback" -> sinkCallback.getClass.getName)
+    s"SnappyStoreSink[${params.mkString(",")}]"
```

Comment:
nit: the separator could be ", " (with a space after ,) for better formatting.

Reply:
Done.